executor/aggregate.go

```diff
@@ -376,22 +372,34 @@ func (w *HashAggPartialWorker) run(ctx sessionctx.Context, waitGroup *sync.WaitG
 		}
 		w.memTracker.Consume(-w.chk.MemoryUsage())
 		waitGroup.Done()
+		w.aggWorkerStats = time.Since(start)
+		if w.stats != nil {
+			atomic.AddInt64(&w.stats.PartialTotalTime, int64(time.Since(start)))
```

Comment:
`PartialTotalTime ` can be collected by sum of `stats.PartialWorkerTime`? so it is redundant?